src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -54,10 +56,18 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.customerOrderList = this.addressBook.getCustomerOrderList();
         this.supplyOrderObservableList = this.supplyOrderList.getOrders();
         this.customerOrderObservableList = this.customerOrderList.getOrders();
+
+        // Initialize catalogues and storage
+        this.ingredientCatalogue = ingredientCatalogue;
+        this.pastryCatalogue = pastryCatalogue;
+        this.storage = storage;
+
+        // Initialize inventory with the ingredient catalogue
+        this.inventory = new Inventory(ingredientCatalogue);
```

Comment:
Should the inventory be initialised with the pastry catalogue and the ingredient catalogue?